Update contributing guide for files under data directory. (#2411)
boto/botocore: CONTRIBUTING.rst

  comment on changes in code coverage for every pull request.
  In practice, this means that every bug fix and feature addition should
  include unit tests.
* If you identify an issue with the JSON service descriptions,
  such as ``botocore/data/aws/s3/2006-03-01/service-2.json``, please submit an
  `issue <https://github.com/boto/botocore/issues>`__ so we can get it
  fixed upstream.

> * Changes to paginators, waiter, and endpoints are also generated upstream based on our internal knowledge of the AWS services.

Reviewer: *waiters

> * Changes to paginators, waiter, and endpoints are also generated upstream based on our internal knowledge of the AWS services.

Author: ```suggestion
* Changes to paginators, waiters, and endpoints are also generated upstream based on our internal knowledge of the AWS services.
```

  These include any of the  following files in ``botocore/data/``:
  
  * ``_endpoints.json``
  * ``*.paginators-1.json``
  * ``*.waiters-2.json``

>   If there is something incorrect with or missing from these files, such as a missing paginator or waiter, please submit an

Reviewer: I'd suggest simplifying the first part of this sentence to: "if you identify an issue with", similar to the sentence on line 30.

>   If there is something incorrect with or missing from these files, such as a missing paginator or waiter, please submit an

Author: ```suggestion
  If you identify an issue with these files, such as a missing paginator or waiter, please submit an
```

  comment on changes in code coverage for every pull request.
  In practice, this means that every bug fix and feature addition should
  include unit tests.
* If you identify an issue with the JSON service descriptions,
  such as ``botocore/data/aws/s3/2006-03-01/service-2.json``, please submit an
  `issue <https://github.com/boto/botocore/issues>`__ so we can get the issue
  fixed upstream.
* Changes to paginators, waiter, and endpoints are also generated upstream based on our internal knowledge of the AWS services.
  These include any of the  following files in ``botocore/data/``:
  
  * ``_endpoints.json``
  * ``*.paginators-1.json``
  * ``*.waiters-2.json``

  If there is something incorrect with or missing from these files, such as a request for a missing paginator or waiter, please submit an
  `issue <https://github.com/boto/botocore/issues>`__ so we can get the issue
  fixed upstream.

> *  We do accept, and encourage, changes to the ``botocore/data/_retry.json`` file.

Author: Is this still true?
